### PR TITLE
Limit stripe_subplat access and display tile

### PR DESCRIPTION
**Pull request by dividehex**
This PR narrows access to the Stripe subplat account and displays the Stripe tile on the dashboard

### CODE REVIEW

**Comment by dividehex** [comment]
@gcoxmoz Would you mind verifying the LDAP groups listed in this PR?  I noticed a typo in the bug comment I copied them from so I just want to be sure the typo wasn't copied over to LDAP also.

File: apps.yml [hunk 1]
```diff
@@ -3712,12 +3712,14 @@ apps:
     url: https://stats.voice.mozit.cloud/
 - application:
     authorized_groups:
-    - team_moco
-    - team_mofo
+    - stripe_subplat_admin
+    - stripe_subplat_analyst
+    - stripe_subplat_developer
+    - stripe_subplat_supportsp
```
Comment (gcoxmoz) [inline apps.yml:3718]: stripe_subplat_viewonly was added in https://bugzilla.mozilla.org/show_bug.cgi?id=1637117#c33 so I think you want that here also?

**Review by gcoxmoz — CHANGES_REQUESTED** [review]

File: apps.yml (hunk 1 above)
Comment (dividehex) [reply apps.yml:3718]: @gcoxmoz Thanks for catching that! 💯

**Review by gcoxmoz — APPROVED** [review]
Group list makes sense, display change makes sense.  Based on the comments at the top of the file I think this does what you expect.  I have no reason to think this is a bad change.

**Pull request merged**